ansible/roles/commcarehq/templates/localsettings.py.j2

```diff
@@ -252,6 +252,7 @@ SEND_BROKEN_LINK_EMAILS = True
 
 OVERRIDE_LOCATION = "https://www.commcarehq.org"
 DEFAULT_PROTOCOL = "https"
+SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTOCOL', 'https')
```

Comment:
I think it is HTTP_X_FORWARDED_PROTO

Reply:
It doesn't matter what it's called as long as it matches up with what we put in nginx